asapdiscovery-alchemy/asapdiscovery/alchemy/cli/prep.py

```diff
@@ -196,15 +196,26 @@ def run(
     )
     console.print(message)
 
+    # create a csv of the failed ligands with the failure reason
     if alchemy_dataset.failed_ligands:
+        rows = []
+        failed_ligands_file = output_folder.joinpath("failed_ligands.csv")
         message = Padding(
-            f"[yellow]WARNING some ligands failed to have poses generated see failed_ligands files in [repr.filename]{output_folder}[/repr.filename][/yellow]",
+            f"[yellow]WARNING some ligands failed to have poses generated see [repr.filename]{failed_ligands_file}[/repr.filename][/yellow]",
```

Comment:
can we not just set `f"[yellow]WARNING {len(alchemy_dataset.failed_ligands )} ligands failed to have [..]"`?

Reply:
Yup great idea!